app/ApiObjects/Comment.php

```diff
@@ -0,0 +1,39 @@
+<?php namespace App\ApiObjects;
+
+use App\ApiObjects\User as ApiUser;
+use Illuminate\Contracts\Support\Arrayable;
+use Illuminate\Contracts\Support\Jsonable;
+use Illuminate\Support\Facades\Auth;
+
+class Comment implements Arrayable, Jsonable
+{
+    protected $comment;
+
+    public function __construct($comment)
+    {
+        $this->comment = $comment;
+    }
+
+    public function toJson($options = 0)
```

Comment:
I know why you did that! The method has to be compatible with that of its parent class `Illuminate\Contracts\Support\Jsonable::toJson()`. Spits out errors if that `$options` parameter isn't there.